src/main/java/com/easysplit/ess/transactions/domain/models/DebtEntity.java

```diff
@@ -14,6 +14,7 @@
  */
 public class DebtEntity {
     private String debtGuid;
+    private String transactionGuid;
```

Comment:
Let's remove this property is we not need it, we can added if for some reason having this property here is helpfull, for now it just seems redundant